client/types/types.go

```diff
@@ -94,7 +94,8 @@ func (o *Operation) Equal(o2 *Operation) error {
 }
 
 func (o *Operation) Filename() (filename string) {
-	filename = fmt.Sprintf("dkg_id_%s", o.DKGIdentifier[:5])
+	shortDkgID := strings.Replace(o.DKGIdentifier, "dkg_identifier_", "", 1)[:5]
```

Comment:
Isn't o.DKGIdentifier just a hex string? (like `f330858c499d1c880b1dbd65c2657440ace3dc4e`). Why do we need this replace call here?

Reply:
Name is going to be too long, i don't think we need the full id

Reply:
Yeah, that is why [:5] is present here - to shorten the id. I don't get why replace is called.

Reply:
ah, right